RecoLocalCalo/EcalRecProducers/plugins/alpaka/AmplitudeComputationKernels.h

```diff
@@ -0,0 +1,37 @@
+#ifndef RecoLocalCalo_EcalRecProducers_plugins_alpaka_AmplitudeComputationKernels_h
+#define RecoLocalCalo_EcalRecProducers_plugins_alpaka_AmplitudeComputationKernels_h
+
+#include "CondFormats/EcalObjects/interface/alpaka/EcalMultifitConditionsDevice.h"
+#include "DataFormats/EcalDigi/interface/alpaka/EcalDigiDeviceCollection.h"
+#include "DataFormats/EcalRecHit/interface/alpaka/EcalUncalibratedRecHitDeviceCollection.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/traits.h"
+#include "DeclsForKernels.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  namespace ecal {
+    namespace multifit {
+
+      namespace v1 {
```

Comment:
here I'm just curious... why a `v1` namespace ?

Reply:
This comes from the CUDA implementation https://github.com/cms-sw/cmssw/blob/master/RecoLocalCalo/EcalRecProducers/plugins/AmplitudeComputationKernels.h#L14 
Someone must have thought that there may be a second version at some point.
I do not think this is going to happen any time soon though.

Reply:
Removed the namespace.